packages/oc-azure-storage-adapter/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "oc-azure-storage-adapter",
+  "version": "1.0.0",
```

Comment:
Let's put 0.0.1, we will set the version during the first lerna publish

Reply:
done :)